Explorer/Assets/DCL/Input/Component/InputMapComponent.cs

```diff
@@ -56,14 +56,15 @@ public InputMapComponent(Kind flags)
         public void BlockInput(Kind kind)
         {
             inputBlockCounters[kind] += 1;
+            ReportHub.Log(ReportCategory.INPUT, $"Block {kind} - {inputBlockCounters[kind]}");
```

Comment:
Only to be sure: is this log for debugging purposes? or do we want it in the final code?

Reply:
TBH, its for both? but as we strip the logs from prod builds, I chose to leave it, as its really useful when we need to debug input issues and I have to re-add it every time I need to do so :S. And we can filter it out using the ReportCategories... wdyt?